erpnext/controllers/website_list_for_contact.py

```diff
@@ -45,17 +44,42 @@ def get_transaction_list(doctype, txt=None, filters=None, limit_start=0, limit_p
 		filters.append((doctype, key, "in", parties))
 
 		if key:
-			return post_process(doctype, get_list(doctype, txt,
+			return post_process(doctype, get_trans_data_list(doctype, txt,
 				filters=filters, fields = "name",
 				limit_start=limit_start, limit_page_length=limit_page_length,
 				ignore_permissions=True,
 				order_by = "modified desc"))
 		else:
 			return []
 
-	return post_process(doctype, get_list(doctype, txt, filters, limit_start, limit_page_length,
+	return post_process(doctype, get_trans_data_list(doctype, txt, filters, limit_start, limit_page_length,
 		fields="name", order_by = "modified desc"))
 
+def get_trans_data_list(doctype, txt, filters, limit_start, limit_page_length=20, ignore_permissions=False,fields=None, order_by=None):
```

Comment:
Ok sure @rmehta , I'll make the changes and make a simple test case. Thanks! I'll make update once ready.